Automattic/wp-calypso
# Launchpad: Use the Wired Launchpad component on the Customer Home

Related to #82669

## Proposed Changes

* As part of the breakdown of #82669 we need to update the Customer Home to use the `DefaultWiredLaunchpad` component instead of the `Launchpad`. The new `DefaultWiredLaunchpad` was created to centralize the basic functionalities around adding the Launchpad to various contexts.

## Testing Instructions



* We need to test the Launchpad on the Customer Home, which also involves the Pre-launch(when skipping the Launchpad on the onboarding)

* Use the Calypso Live link below or apply this PR to your local environment
* Create a new site with the build intent
* Go through the onboarding flow until you reach the Fullscreen Launchpad step
* Skip it, and you should be redirected to the Customer Home
* Go through the tasks and Launch the site. Make sure it works as expected.
* You then should see the Post-launch launchpad.
* Complete all the tasks again and make sure you can dismiss the Launchpad.

## Pre-merge Checklist
